gno.land/cmd/gnoland/README.md

```diff
@@ -6,8 +6,88 @@
     $> cd ./gno/gno.land
     $> make install.gnoland
 
-## Run `gnoland` full node
+## Option 1: Run `gnoland` full node for local development
 
     $> gnoland start
 
 Afterward, you can interact with [`gnokey`](../gnokey) or launch a [`gnoweb`](../gnoweb) interface.
+
+
+## Option 2: Run a node and sync with a Proof of Authority (POA) network
+
+ - gnoland init
```

Comment:
I believe this step is unnecessary in option 2.

Reply:
Yes, it's needed to generate the $DAEMON_HOME (GNO_HOME) directory, with the config and data/